src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -597,6 +602,13 @@ public String bindAddress() {
     return bindAddress;
   }
 
+  @Override
+  public FilenameMaker getFilenameTemplate() {
```

Comment:
The name of this method and what it actually returns are not consistent with each other. It might make sense to return the template string and initialise the filename maker somewhere common, rather than doing it inside the `Options` impls.